ToolkitComponent.SourceProject.props

```diff
@@ -17,6 +17,7 @@
 
   <PropertyGroup>
     <IsAotCompatible Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))">true</IsAotCompatible>
```

Comment:
This feels weird. Where is the target framework being set? This is a .props file, so it's imported before the .csproj. Are we _sure_ that this can actually see the correct `TargetFramework` value and it's not imported too early?

Reply:
Yes, TFMs are set by the MultiTarget tooling on [line 7 of this file](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/212/files/79bd7eba78747d40f4919b7e170e771176c11105#diff-538f1da3c5e09cc2e9c6be9073c66700fc6b57c8027fb18dba0435bd12ccac89R7).